project.clj

```diff
@@ -56,7 +56,8 @@
                       :dependencies [[org.clojure/clojure "1.8.0-master-SNAPSHOT"]]}
 
              :test-clj {:test-paths ["test/clj"]
-                        :java-source-paths ["test/java"]}
+                        :java-source-paths ["test/java"]
+                        :resource-paths ["test/resources"]}
```

Comment:
Why is this needed? I don't see anything related in the PR (guess I might be missing something).